src/styles/blocks/container.scss

```diff
@@ -0,0 +1,28 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+
+  &__block {
+    height: 300px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    color: #fff;
+  }
+
+  $items:
+    "1" rgb(100%, 0, 0),
+    "2" rgb(80%, 0, 0),
+    "3" rgb(60%, 0, 0),
+    "4" rgb(40%, 0, 0),
+    "5" rgb(20%, 0, 0),
+    "6" rgb(0, 0, 0);
+
+  @each $number, $color in $items {
+    &__block--#{$number} {
+      background-color: $color;
+    }
+  }
```

Comment:
If you using SASS u need to follow the nesting rules, namely set this style in &__block styles.